### PR TITLE
AN-5517/api integration - GET minting/assets

### PR DESCRIPTION
**Pull request by forgxyz**
External Table Config: https://github.com/FlipsideCrypto/streamline-snowflake/pull/189

### CODE REVIEW

File: models/streamline/external/storefront_items/streamline__minting_assets_realtime.sql [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        func = '{{this.schema}}.udf_bulk_rest_api_v2',
+        target = "{{this.schema}}.{{this.identifier}}",
+        params ={ "external_table": "minting_assets",
+        "sql_limit": "1",
+        "producer_batch_size": "1",
+        "worker_batch_size": "1",
+        "sql_source": "{{this.identifier}}" }
+    )
+) }}
+
```
Comment (eric-laurello) [inline models/streamline/external/storefront_items/streamline__minting_assets_realtime.sql:14]: got a dependency issue when running - needs to add:

` -- depends_on: {{ ref('silver_api__storefront_items') }}`

File: models/streamline/external/storefront_items/streamline__minting_assets_realtime.sql [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+{% if not var(
+        'STOREFRONT_INITIAL_RUN',
+        false
+    ) %}
+    {% if execute %}
+        {% set query %}
+            WITH target_entry_id AS (
+
+                SELECT
+                    entry_id,
```
Comment (eric-laurello) [inline models/streamline/external/storefront_items/streamline__minting_assets_realtime.sql:24]: is this the correct column? it's not in the silver table

Comment (eric-laurello) [reply models/streamline/external/storefront_items/streamline__minting_assets_realtime.sql:24]: maybe item_id?

File: models/streamline/external/storefront_items/streamline__minting_assets_realtime.sql (hunk 1 above)
Comment (forgxyz) [reply models/streamline/external/storefront_items/streamline__minting_assets_realtime.sql:14]: Ty

File: models/streamline/external/storefront_items/streamline__minting_assets_realtime.sql (hunk 2 above)
Comment (forgxyz) [reply models/streamline/external/storefront_items/streamline__minting_assets_realtime.sql:24]: Whoops, correct it needs to be updated

**Review by eric-laurello — APPROVED** [review]

**Pull request merged**